src/OSPSuite.Infrastructure.Import/Core/DataFormat/MixColumnsDataFormat.cs

```diff
@@ -39,6 +44,7 @@ protected override UnitDescription ExtractUnits(string description, IUnformatted
                .Substring(1, units.Length - 2) //remove the brackets
                .Trim()                         //remove whitespace
                .Split(',')                     //split comma separated list
+               .Where(unitName => _dimensionFactory.Dimensions.SelectMany(dimension => dimension.Units).Any(unitFromDimension => unitFromDimension.Name == unitName)) //only accepts valid units
```

Comment:
I do not think this is correct here. We have a lot of methods checking whether a unit is valid or not (taking care of case, synonyms etc...) 
This needs to change